DTAdmin/Group/GroupDetailsViewController.swift

```diff
@@ -24,6 +24,15 @@ class GroupDetailsViewController: UIViewController {
     @IBAction func getTimeTableByGroupTapped(_ sender: Any) {
     }
     @IBAction func getResultsByGroupTapped(_ sender: Any) {
+      guard let resultByGroupViewController = UIStoryboard(name: "Result", bundle: nil).instantiateViewController(withIdentifier: "ResultByGroup") as? ResultByGroupViewController else  { return }
+      
+//      guard let test = tests?[indexPath.row] else {
```

Comment:
Please remove commented dead code or provide good justification why have you commented it.